news/2023/2023-10-08-osu-mania-world-cup-2023.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2023 Concludes"
+date: 2023-10-08 21:00:00 +0000
+---
+The osu!mania 4K World Cup concluded as fast as it started. Get up to speed with the tournament's most memorable highlights!
+
+It is with great pleasure that the 10th edition of the osu!mania 4K World Cup has met its expectations. Since its first edition in [2014](/wiki/Tournaments/MWC/2014), users all across the globe have been able to showcase the best of the best players their countries were able to produce! Ever since then, the game and community has evolved quite a lot. The 10th edition of the tournament is a culmination of everyone's efforts.
+
+Reminscing aside, lets recall the present shall we? We would like to use this opportunity to focus on the players of today thanking the efforts of the community of yesteryear!
+
+![](/wiki/Tournaments/MWC/2023_4K/img/mwc4k2023-banner.jpg)
+
+## Summary
+
+![](/wiki/shared/news/2023-10-08-osu-mania-world-cup-2023/mwc4k2023_podium.jpg)
+
+First and foremost, congratulations to the United States of America for standing atop the podium! This is the second time they managed to reach the coveted 1st place spot since 2015. Suffice to say, the team - although with different rosters - has participated ever since the tournament's humble beginnings. This feat cannot be understated, and despite the multiple frustrations reaching 2nd and 3rd place multiple times, they managed to snatch the gold medal from everyone else!
+
+Why is this a big feat you may ask? Getting 1st place isn't easy, that much is obvious. Meeting the former 4K Champion after being bested in the Finals in the tournament's decisive Grand Finals match, *now that's scary~*. For those who missed it, we suggest [checking the VOD of the match](https://www.twitch.tv/videos/1933741044) at your own leisure!
```

Comment:
```suggestion
Why is this a big feat, you may ask? Getting first place isn't easy, that much is obvious; but meeting the former 4K World Champion in the tournament's decisive Grand Finals match? *Now that's scary~*. For those who missed it, we suggest [checking the VOD of the match](https://www.twitch.tv/videos/1933741044) at your own leisure!
```

Reply:
US won against Korea in the Finals match, current wording implies otherwise i think.